packages/bot/src/handlers/invites/INVITE_CREATE.ts

```diff
@@ -2,5 +2,6 @@ import type { DiscordGatewayPayload, DiscordInviteCreate } from '@discordeno/typ
 import type { Bot } from '../../index.js'
 
 export async function handleInviteCreate(bot: Bot, data: DiscordGatewayPayload): Promise<void> {
-  bot.events.inviteCreate?.(bot.transformers.invite(bot, data.d as DiscordInviteCreate))
+  const invite = bot.transformers.invite(bot, data.d as DiscordInviteCreate)
```

Comment:
Undo the change in this file

Reply:
its better without the need of the const

Reply:
Okkay